data/languages/gdscript.yaml

```diff
@@ -0,0 +1,81 @@
+            # spatial
+            force_update_transform get_parent_spatial get_world global_rotate global_scale global_translate hide is_local_transform_notification_enabled is_scale_disabled is_set_as_toplevel is_transform_notification_enabled is_visible_in_tree look_at look_at_from_position orthonormalize rotate rotate_object_local rotate_x rotate_y rotate_z scale_object_local set_as_toplevel set_disable_scale set_identity set_ignore_transform_notification set_notify_local_transform set_notify_transform show to_global to_local translate translate_object_local update_gizmo
+            # node
+            add_child add_child_below_node add_to_group can_process duplicate find_node find_parent get_child get_child_count get_children get_groups get_index get_network_master get_node get_node_and_resource get_node_or_null get_parent get_path get_path_to get_physics_process_delta_time get_position_in_parent get_process_delta_time get_scene_instance_load_placeholder get_tree get_viewport has_node has_node_and_resource is_a_parent_of is_displayed_folded is_greater_than is_in_group is_inside_tree is_network_master is_physics_processing is_physics_processing_internal is_processing is_processing_input is_processing_internal is_processing_unhandled_input is_processing_unhandled_key_input move_child print_stray_nodes print_tree print_tree_pretty propagate_call propagate_notification queue_free raise remove_and_skip remove_child remove_from_group replace_by request_ready rpc rpc_config rpc_id rpc_unreliable rpc_unreliable_id rset rset_config rset_id rset_unreliable rset_unreliable_id set_display_folded set_network_master set_physics_process set_physics_process_internal set_process set_process_input set_process_internal set_process_priority set_process_unhandled_input set_process_unhandled_key_input set_scene_instance_load_placeholder
+            # object
+            callv can_translate_message connect disconnect emit_signal free get get_class get_incoming_connections get_indexed get_instance_id get_meta get_meta_list get_method_list get_property_list get_script get_signal_connection_list get_signal_list has_meta has_method has_user_signal is_blocking_signals is_class is_connected is_queued_for_deletion notification property_list_changed_notify set set_block_signals set_deferred set_indexed set_message_translation set_meta set_script tr
+        - > # signals
+            script_changed ready renamed tree_entered tree_exited tree_exiting visibility_changed input_event mouse_entered mouse_exited area_entered area_exited area_shape_entered area_shape_exited body_entered body_exited body_shape_entered body_shape_exited draw hide item_rect_changed joy_connection_changed
+comments:
+    line: "#"
+    # block: ['"""', '"""']
+styles:
+    0: 32 # default white-space
+    1: 126 # comment
+    2: 113 # number
+    3: 104 # string
+    4: 106 # character
+    5: 165 # keywords 0 (for keywords)
```

Comment:
This is style for "deleted" line, which makes no sense. Please retain styles semantic meaning.